pymtl3/passes/PassGroups.py

```diff
@@ -43,8 +45,11 @@
   DynamicSchedulePass(),
   CLLineTracePass(),
   VcdGenerationPass(),
+  #WaveGenPass(),
```

Comment:
If `WaveGenPass` is not critical to the functionality of this PR, we probably should remove this?